Makefile

```diff
@@ -18,40 +17,10 @@ help: ## Shows all targets and help from the Makefile (this message).
 			} \
 		}'
 
-## Linters
-#####################################################################
-
-lint: ## Run all linters.
-lint: golangci-lint yamllint
-
-golangci-lint: ## Runs the golangci-lint linter.
-	@set -e;\
-		extraargs=""; \
-		if [ "$(OUTPUT_FORMAT)" == "github" ]; then \
-			extraargs="--out-format github-actions"; \
-		fi; \
-		$(GOBIN)/golangci-lint run -c .golangci.yml ./... $$extraargs
-
 yamllint: ## Runs the yamllint linter.
```

Comment:
In a later PR, probably worth removing the Makefile entirely and moving this block into `.github/workflows/lint.yaml`